tests/ui-internal/allow_incorrect_lint_formulations.rs

```diff
@@ -0,0 +1,22 @@
+#![allow(clippy::almost_standard_lint_formulation)]
```

Comment:
I would like this attribute to be more fine-grained, we'll likely only want to enable it on a lint by lint level, and not on a module level like here. Can you try attaching it to the lint name directly:

```
#[allow(clippy::almost_standard_lint_formulation)]
#[clippy::version = "pre 1.29.0"]
pub clippy::ALLOWED_INVALID,
```

then you can also move it into the same test file as the other tests :)